packages/components/src/date-time/style.scss

```diff
@@ -54,19 +54,35 @@
 		font-size: $default-font-size;
 	}
 
+	// Seperate borders so that border respect border radius
+	.CalendarMonth_table {
+		border-collapse: separate;
+		border-spacing: 2px;
+	}
+
 	.CalendarDay {
 		font-size: $default-font-size;
-		border: 1px solid transparent;
+		border: none;
 		border-radius: $radius-round;
 		text-align: center;
+
+		&:focus {
+			box-shadow: inset 0 0 0 $border-width-focus theme(primary), inset 0 0 0 #{ $border-width-focus + $border-width } $white;
+			outline: 2px solid transparent; // Shown in Windows 10 high contrast mode.
+		}
 	}
 
 	.CalendarDay__selected {
 		background: theme(primary);
+		border: 2px solid transparent; // This indicates selection in Windows 10 high contrast mode.
```

Comment:
On a selected day, we explicitly paint a solid, but transparent border. This will show up as a round border in Windows 10 high contrast mode. 

But the outline will also show up. 

So focus in Windows 10 high contrast mode is a square rectangle, and seleted is a round circle. They stack:

<img width="61" alt="Screenshot 2020-04-15 at 11 02 38" src="https://user-images.githubusercontent.com/1204802/79319541-be749a00-7f08-11ea-9654-b0ae75c90213.png">